src/components/Chip.tsx

```diff
@@ -57,4 +61,5 @@ const typeStyles: Record<ChipType, Properties> = {
   dark: Css.bgGray900.white.$,
   neutral: Css.bgGray200.$,
   darkMode: Css.bgGray700.white.$,
+  info: Css.bgLightBlue100.$,
```

Comment:
@bdow i think it would be easier for us to just add the new "info" type directly to the chipType (w/ how we need to use it on the FE as a status tag)

let me know if you want me to remove the "color" | "backgroundColor" xss update or keep it there as an escape hatch for other uses

Reply:
Yea, that's fine with me. 
If we add it as a supported type here, then we should also include it in the Beam Figma. Can you work with your designer to get it added if you decide to go this route?

Reply:
Thanks, @bdow yes will work with Radhi to update it beam figma 👍 

would you like me to remove the "color" | "backgroundColor" xss change or keep that?

Reply:
We can keep it. I think it'll be nice to have that escape hatch if we really need a one-off. Thanks!

Reply:
Just realized you probably grabbed "info" from the Tag component with the same background-color, maybe? I'd say feel free to merge this as we already have a precedent for a `type="info"` with this color defined. 👍

Reply:
Ok sg, yes I did grab "info" from the Tag component as a precedent 👍